tools/tpm2_tool.c

```diff
@@ -134,6 +135,8 @@ int main(int argc, char *argv[], char *envp[]) {
     /* TODO SAPI INIT */
     TSS2_SYS_CONTEXT *sapi_context = sapi_ctx_init(tcti);
 
+    tpm2_errata_init(sapi_context);
```

Comment:
Maybe this function should be called by sapi_ctx_init()?

Reply:
The semantics of sapi init is simple and atomic, so this is why I place errata_int outside it. But I don't mind moving it to sapi init.